lxd/networks.go

```diff
@@ -223,4 +223,48 @@ func networkDelete(d *Daemon, r *http.Request) Response {
 	return EmptySyncResponse
 }
 
-var networkCmd = Command{name: "networks/{name}", get: networkGet, delete: networkDelete}
+func networkPost(d *Daemon, r *http.Request) Response {
+	name := mux.Vars(r)["name"]
+	req := shared.NetworkConfig{}
+
+	// Parse the request
+	err := json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		return BadRequest(err)
+	}
+
+	// Get the existing network
+	_, dbInfo, _ := dbNetworkGet(d.db, name)
```

Comment:
This pattern seems a little weird to me. Why not just use the error from dbNetworkGet like we do everywhere else?